spec/features/transactions_filter_spec.rb

```diff
@@ -27,7 +27,7 @@
 
     before do
       visit root_path
-      click_on 'Filters'
+      find(:css, 'a#toggle_filter').trigger('click')
```

Comment:
@yunixon you may still reference link by title. So `click_on 'Filters'` should work.